scaladoc/src/dotty/tools/scaladoc/Scaladoc.scala

```diff
@@ -95,9 +95,6 @@ object Scaladoc:
 
     allSettings.filterNot(scaladocSpecificSettings.contains).foreach(setInGlobal)
 
-    summary.warnings.foreach(report.warning(_))
```

Comment:
Why do we want to remove that?

Reply:
It was redundantly called by `checkUsage` so we had every warning reported two times.

Reply:
will that work with cmd?

Reply:
Will check

Reply:
Yes, it will